AIMSICD/src/main/res/values/translatable_strings.xml

```diff
@@ -318,7 +318,7 @@
 
 
     <!-- Section Main -->
-    <string name="help_main_neighboring_cells">Shows neighboring cells.</string>
+    <string name="help_main_neighboring_cells">Shows neighbouring cells.</string>
```

Comment:
Please change that back as well. I don't see a reason why we should do this at all, we should stick with one word.